NetworkChat/Client.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Net;
+using System.Net.Sockets;
+using System.IO;
+
+
+namespace NetworkChat
```

Comment:
```suggestion
using System.IO;

namespace NetworkChat
```
Две пустые строки подряд обычно вообще не ставятся.